tools/accuracy_checker/accuracy_checker/annotation_converters/super_resolution_converter.py

```diff
@@ -71,10 +84,31 @@ def parameters(cls):
 
     def configure(self):
         self.data_dir = self.get_value_from_config('data_dir')
+        self.lr_dir = self.get_value_from_config('lr_dir') if self.get_value_from_config('lr_dir') else self.data_dir
+        if not self.lr_dir:
+            raise ConfigError('One of the parameters: data_dir or lr_dir should be provided for conversion')
+        self.hr_dir = self.get_value_from_config('hr_dir') if self.get_value_from_config('hr_dir') else self.data_dir
+        self.upsampled_dir = self.get_value_from_config('upsampled_dir')
+        if not self.upsampled_dir:
+            self.upsampled_dir = self.lr_dir
         self.lr_suffix = self.get_value_from_config('lr_suffix')
         self.hr_suffix = self.get_value_from_config('hr_suffix')
         self.upsample_suffix = self.get_value_from_config('upsample_suffix')
         self.two_streams = self.get_value_from_config('two_streams')
+        if self.two_streams:
+            if not self.data_dir:
+                if self.lr_dir == self.upsampled_dir:
+                    self.data_dir = self.lr_dir
+                else:
+                    raise ConfigError('data_dir parameter should be provided for conversion as common part of paths '
```

Comment:
why here can not be specified relative to lr_dir?

Reply:
there is case when path relative to lr_dir can not be found.
Example: lr_dir = common_path/LR, upsampled_dir = common_path/UP

Reply:
your example is not issue
upsample_dir = lr_dir/../UP